packages/sakuli-legacy/src/context/sahi/fetch/fetch-api.interface.ts

```diff
@@ -1,28 +1,369 @@
+     *
+     * @example
+     * Assume the following HTML-snippet:
+     * ```html
+     * <div id="d1" style="background-color: yellow"><i>Formatted</i> Text</div>
+     * ```
+     *
+     * ```typescript
+     * await _containsHTML(_div(0), '<i>Formatted</i>'); // -> true
+     * await _containsHTML(_div(0), 'Text'); // -> true
+     * await _containsHTML(_div(0), '<i>Formatted</i> Text'); // -> true
+     * await _containsHTML(_div(0), '<i>.*</i>'); // -> true
+     * await _containsHTML(_div(0), 'Formatted Text'); // -> false
+     * await _containsHTML(_div(0), 'Non existent'); // -> false
+     * ```
+     *
+     * @param query
+     * @param html
+     */
     _containsHTML(query: SahiElementQueryOrWebElement, html: string): Promise<boolean>;
+
+    /**
+     * Check whether an _parent_ element contains a _child_ element.
+     *
+     * @example
+     * Assume the following HTML-snippet:
+     * ```html
+     * <div id="parent">
+     *     <div id="child-a"></div>
+     *     <div id="child-b"></div>
+     * </div>
+     * ```
+     *
+     * ```typescript
+     * await _contains(_div('parent'), _div('child-b')) // -> true
+     * await _contains(_div('child-a'), _div('child-b')) // -> true
+     * ```
+     *
+     * @param parent
+     * @param child
+     */
     _contains(parent: SahiElementQueryOrWebElement, child: SahiElementQueryOrWebElement): Promise<boolean>
+
+    /**
+     * Retrieves the current text-content of the documents [`title`]{@link https://developer.mozilla.org/de/docs/Web/HTML/Element/title} element.
+     *
+     * @example
+     * Assuming the following document:
+     * ```html
+     * <!DOCTYPE html>
+     * <html>
+     * <head>
+     *   <title>Tested with Sakuli</title>
+     * </head>
+     * <body>
+     * </body>
+     * </html>
+     * ```
+     *
+     * ```typescript
+     * const title = await _title() // -> "Tested with Sakuli";
+     * ```
+     */
     _title(): Promise<string>;
+
+    /**
+    * Styles in HTML elements are calculated by the browser based on various CSS rules.
+    * _style returns the computed style that is finally applicable to the element.
+    * If the element inherits the named style from
+    * its parent, the parent will be queried for its value.  Where possible,
+    * color values will be converted to their hex representation (e.g. #00ff00
+    * instead of rgb(0, 255, 0)).
+    *
+    * Accessing style directly as an attribute will not give computed style.
+    * Always use _style instead.
+    *
+    * _Warning:_ the value returned will be as the browser interprets it, so
+    * it may be tricky to form a proper assertion.
+    *
+    * @param query
+    * @param attr
+    */
     _style(query: SahiElementQueryOrWebElement, attr: string): Promise<string>;
+
+    /**
+     * Returns a tuple describing an element's location, in pixels relative to
+     * the document element.
+     *
+     * @example
```

Comment:
The formating doesn't look quite right as the `example` is in the code box.
![example](https://user-images.githubusercontent.com/48590838/66577250-50c26c80-eb79-11e9-8323-510ee3a3fce6.png)